Gemfile

```diff
@@ -15,3 +15,6 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
Oh, then that works for me.  Since we don't have a Gemfile.lock in version controller, we don't even need to edit that.